editor/test/integration/engine/native_extensions_test.clj

```diff
@@ -123,32 +123,29 @@
           test-prefs (test-util/make-test-prefs)]
       (assert (= (native-extensions/get-build-server-url test-prefs) "https://build-stage.defold.com"))
       (testing "clean project builds on server"
-        (g/with-auto-evaluation-context evaluation-context
```

Comment:
All of this is just whitespace changes from removing the `g/with-auto-evaluation-context` wrappers.